app/code/Magento/Theme/view/frontend/web/js/cookie-status.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'jquery',
+    'mage/translate',
```

Comment:
Just move this dependency as the last one, so you don't need to introduce `$tr` which is not used anyway.

Reply:
Done